INSTALLFEST.md

```diff
@@ -266,411 +164,80 @@ brew install nvm
-
--   Generate a new key by running
-
-    ```bash
-    ssh-keygen -t rsa -C "<your_email@example.com>"
-    ```
-
-(Feel free to put in a password or select a non-default location for your keys,
-but it's not necessary to do so; to move ahead, just keep hitting `enter`).
-
--   Add this new key to your system by running:
-
     ```bash
-    ssh-add ~/.ssh/id_rsa
+    config/npm.sh
     ```
 
--   Copy the new key to your clipboard using either:
-
-    ```bash
-    # OSX ONLY
-      pbcopy < ~/.ssh/id_rsa.pub
-    ```
-
-    **OR**
-
-    ```bash
-    # LINUX ONLY
-      xclip -selection clipboard < ~/.ssh/id_rsa.pub
-    ```
-
--   Log into GitHub.com, go to [https://github.com/settings/ssh](https://github.com/settings/ssh),
-    and paste in your SSH key. To test it out, type the following into the command
-    line:
-
-    ```bash
-    ssh -T git@github.com
-    ```
-
-    If you get a prompt along the lines of
-
-    ```bash
-    The authenticity of host 'github.com (xxx.xxx.xxx.xxx)'... can\'t be established.
-    RSA key fingerprint is XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX:XX.
-    Are you sure you want to continue connecting (yes/no)?
-    ```
-
-    Just type 'yes'. If everything's working, you should get a response like the
-    following:
-
-    ```bash
-    Hi yourUsername! You\'ve succesfully authenticated, but GitHub does not provide shell access.
-    ```
-
-Last thing, now that you have git and GitHub we want this repo on your local
-computer. Please follow along as I show you how to fork, clone and put the repo
-in the correct directory.
-
--   Now that our local machine is set up with Git, we need to `fork and clone`
-    the [orientation repo](https://github.com/ga-wdi-boston/orientation). Once you
-    fork to your Github  account, make sure you copy the HTTPS clone link (It will
-    look something like `https://github.com/<your github name>/orientation.git`)
-
--   In your root directory `cd ~`, let's move to our downloads file by `cd
-    Downloads/`, then run `git clone <link copied from github>`.
+## Atom
 
--   Move into the `oritentation` directory by `cd orientation/`. Now we have to set up
-    a global 'excludesfile', listing all the files that we might want git to ignore.
+The text editor we'll be using in this course is called **Atom**; it was
+developed by the GitHub team, and is highly extensible.
 
-    ```bash
-    git config --global core.excludesfile ~/.gitignore
-    cp .gitignore ~/.gitignore # from this repository directory
-    ```
+***Note: From this point forward you will open Atom from the command line***
 
-### Install [`hub`](https://github.com/github/hub)
+### OSX ONLY
 
-> hub is a command line tool that wraps git in order to extend it with extra
-> features and commands that make working with GitHub easier. -- [`hub`
-> README](https://github.com/github/hub).
+1.  Download Atom from [Atom.io](https://atom.io/).
+1.  Move Atom from downloads to the Application directory.
```

Comment:
This step is very important. Without it, the command line tool installation will claim to work when it does not actually work. This is due to sandboxing on macOS.